src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -538,10 +530,12 @@ impl<S: S3BackEnd + Sync + Send + 'static, K: KVEngine + 'static> MetaData for S
         target_path: Option<&Path>,
     ) -> DatenLordResult<(Duration, FuseAttr, u64)> {
         // pre-check
-        let (parent_full_path, new_node_attr, fuse_attr) = {
-            let mut cache = self.cache.write().await;
-            let parent_node = Self::create_node_pre_check(parent, node_name, &mut cache)
+        let (parent_full_path, _, fuse_attr) = {
```

Comment:
If we want to remove the `new_node_attr`, just remove it. Don't leave a blank place holder here.